packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -714,7 +715,10 @@ abstract class ResidentRunner {
   ///
   /// Throws an [AssertionError] if [Devce.supportsScreenshot] is not true.
```

Comment:
I think this comment is incorrect now, since it calls `throwToolExit`?

Reply:
Fixed